src/Prefixer.js

```diff
@@ -63,18 +63,19 @@ export function prefix (value, length) {
 		case 8116: case 7059: case 5753: case 5535:
 		case 5445: case 5701: case 4933: case 4677:
 		case 5533: case 5789: case 5021: case 4765:
+			var temp = value
```

Comment:
not ideal, but introducing a `temp` is the best I could figure out for this 🤷‍♂️